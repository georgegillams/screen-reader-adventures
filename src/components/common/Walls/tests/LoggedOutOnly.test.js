import React from 'react';
import { render } from '@testing-library/react';

import { LoggedOutOnly } from '..';

describe('<LoggedOutOnly />', () => {
  it('Should render correctly', () => {
    const { container } = render(<LoggedOutOnly user={null}>Test</LoggedOutOnly>);

    expect(container).toMatchSnapshot();
  });
  it('Should render correctly with user', () => {
    const { container } = render(<LoggedOutOnly user={{}}>Test</LoggedOutOnly>);

    expect(container).toMatchSnapshot();
  });
});
