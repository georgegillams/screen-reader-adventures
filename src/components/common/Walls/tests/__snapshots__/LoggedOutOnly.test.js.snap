// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoggedOutOnly /> Should render correctly 1`] = `
<div>
  Test
</div>
`;

exports[`<LoggedOutOnly /> Should render correctly with user 1`] = `
<div>
  <div
    class="section__outer"
  >
    <h2
      class="section__heading"
      id="you're-logged-in"
    >
      You're logged in
    </h2>
    <span
      class="paragraph"
    >
      <a
        class="text-link text-link--text-link text-link--no-padding text-link--inline"
        href="/account"
      >
        Go to your account.
      </a>
    </span>
  </div>
</div>
`;
